### PR TITLE
Fixed 445: Support ERC-20 Token

### LINKED ISSUE #455: Support ERC-20 Token
1. support in website wallet.
2. support in desktop wallet.
3. support in explorer.
@ququzone works on contract read and call.
@nmhung1210  works on UI / UX on all the 3.
in 3, need to support 
a. decode data in action to show erc20 token transfer
b. show erc20 token balance in explorer page
c. show actions tranferred to a particular wallet page (this is hard.)

### PR DESCRIPTION
**Pull request by nmhung1210**
**What type of PR is this?**
feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #455 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

**Wallet**

image

**Action detail**

image

### CODE REVIEW

File: src/shared/wallet/account-section.tsx [hunk 1]
```diff
@@ -101,69 +170,191 @@ export default class AccountSection extends React.Component<Props, State> {
...
+        columns={columns}
+        pagination={false}
+        showHeader={false}
+      />
+    );
+  }
+
+  public renderWallet = (wallet: Account) => {
+    return (
+      <Card bodyStyle={{ padding: 0, wordBreak: "break-word" }}>
```
Comment (puncsky) [inline src/shared/wallet/account-section.tsx]: keep border-radius consistent with other places

border-radius: 4px;

File: src/shared/wallet/account-section.tsx [hunk 2]
```diff
@@ -101,69 +170,191 @@ export default class AccountSection extends React.Component<Props, State> {
...
+          style={{
+            backgroundColor: colors.primary,
+            color: colors.white,
+            padding: 20
+          }}
+        >
+          <Col>
+            <Icon type="wallet" /> {t("account.wallet")}
+          </Col>
+          <Col onClick={() => this.showCustomTokensForm()}>
```
Comment (puncsky) [inline src/shared/wallet/account-section.tsx]: cursor should be pointer

File: src/shared/wallet/wallet.tsx [hunk 3]
```diff
@@ -145,19 +151,24 @@ class WalletComponent extends PureComponent<Props, State> {
...
+          >
+            <Col xs={24} sm={12} md={15} lg={16}>
               {wallet &&
                 this.renderTabs({
                   address: wallet.address
                 })}
               {!wallet && this.renderNoWallet()}
             </Col>
-            <Col md={6} push={2}>
+            <Col xs={24} sm={12} md={9} lg={8}>
```
Comment (puncsky) [inline src/shared/wallet/wallet.tsx]: Screen Shot 2019-06-04 at 2 02 23 PM

need some margin

**Comment by puncsky** [comment]
can we add a configuration to add preset tokens?

**Comment by puncsky** [comment]
Screen Shot 2019-06-04 at 2 32 18 PM

do not use red color, use grey

**Comment by nmhung1210** [comment]
> can we add a configuration to add preset tokens?

Yes, we can

**Comment by nmhung1210** [comment]
> Screen Shot 2019-06-04 at 2 32 18 PM
> 
> do not use red color, use grey

ok

**Pull request merged**